src/System.Management/pkg/System.Management.pkgproj

```diff
@@ -0,0 +1,17 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <ProjectReference Include="..\ref\System.Management.csproj">
+      <SupportedFramework>netcoreapp2.0</SupportedFramework>
+    </ProjectReference>
+    <ProjectReference Include="..\src\System.Management.csproj" />
+    <InboxOnTargetFramework Include="net45">
```

Comment:
I would have expected the package index to also contain a change about this new library and where is it inboxed.

Reply:
Ops forgot that, also what is the correct version? the smallest possible?

Reply:
do you mean the assembly version? If so, we should use whatever is on desktop.

Reply:
There was already an entry for System.Management in the packages index, but now with the other changes of build configurations, I've been unable to get a clean local build (yet).